dashboard/app/models/plc/enrollment_unit_assignment.rb

```diff
@@ -75,26 +75,23 @@ def focus_area_stage_ids
   def summarize_progress
     summary = []
 
-    categories_for_stage = plc_course_unit.script.lessons.map(&:flex_category).uniq
-
     # If the course unit has an evaluation level, then status is determined by the completion of the focus group modules
     if plc_course_unit.has_evaluation?
-      Plc::LearningModule::MODULE_TYPES.select {|type| categories_for_stage.include?(type)}.each do |flex_category|
-        module_category = flex_category
-        category_name = I18n.t("flex_category.#{module_category}", default: I18n.t('flex_category.required'))
+      plc_course_unit.script.lesson_groups.each do |lesson_group|
+        next unless Plc::LearningModule::MODULE_TYPES.include?(lesson_group.key)
         summary << {
-          category: category_name,
-          status: module_assignment_for_type(flex_category).try(:status) || Plc::EnrollmentModuleAssignment::NOT_STARTED,
-          link: Rails.application.routes.url_helpers.script_path(plc_course_unit.script, anchor: category_name.downcase.tr(' ', '-'))
+          category: lesson_group.localized_display_name,
```

Comment:
Could we leave the custom `category_name` logic in for PLC for now? According to the tests, it seems important that we use the limited set of category names for PLC instead of having script-specific display names.

I would be fine with restoring that logic here, leaving a TODO, and filing a work item for the PLC team to follow up and  generate appropriate translations for all of the PLC scripts.

I suppose the alternative is going ahead and creating all of the appropriate i18n keys for the PLC scripts now.

Reply:
So this actually works correctly outside of the tests and I have special checks to make sure the key value pairs for PLC are correct for these special cases but I can't figure out how to make that work in tests because it relies on the scripts.en.yml file getting populated which is seems like is not happening for the test situation.

Reply:
Talked to @islemaster about this on a video call. There was confusion on if the translations would work on production which was cleared up. This issue is specific to the test and the way scripts and lesson groups are set up in the test. He was on board moving forward with it as is since PLC has future work to pull out this system. So I'm going to merge this change